src/components/Tool/Appeal/Modals/UpdateDonationsModal/DonationTable/DonationTable.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useMemo } from 'react';
```

Comment:
I assume this component is too different from the existing `DonationTable` to reuse it?

Reply:
I think so. It can probably be done, but the component might then become too confusing. Maybe this is something we can flag for post-switch to investigate.

Reply:
I think it's good as-is for now.